pkg/pgmodel/pgx.go

```diff
@@ -220,8 +219,8 @@ func NewPgxIngestorWithMetricCache(c *pgxpool.Pool, cache MetricCache, cfg *Cfg)
 
 // NewPgxIngestor returns a new Ingestor that write to PostgreSQL using PGX
 func NewPgxIngestor(c *pgxpool.Pool) (*DBIngestor, error) {
-	metrics, _ := bigcache.NewBigCache(DefaultCacheConfig())
-	cache := &MetricNameCache{metrics}
+	//TODO what size should we use?
```

Comment:
I think 10k is ok here too.

Reply:
My bigger question for this, and the [series cache](https://github.com/timescale/timescale-prometheus/pull/158/files#diff-25015b1d7c86d8c5fab194419dee6c51R212) is if we should start the cache smaller and grow it to some maximum if the miss-rate is too high. Generally the smallest cache which fits the data should perform better...

Reply:
I suspect the performance impact will be negligible for Metric cache (favoring a bit oversized), but Series cache will probably be very dependent on the size of records--- especially if undersized for large datasets.

Whether or not it can be grown automatically, maybe the default size be configured via CLI Args/ENV Vars?

Reply:
@ksten-ab Yup, we're definitely adding CLI configuration before this PR, _probably_ not dynamic-expansion though